netbench-orchestrator/src/orchestrator/cli.rs

```diff
@@ -0,0 +1,121 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::orchestrator::{OrchError, OrchResult};
+use serde::Deserialize;
+use std::{
+    fs::File,
+    path::{Path, PathBuf},
+};
+
+#[derive(Clone, Debug)]
+pub struct OrchestratorConfig {
+    // netbench
+    netbench_scenario_filename: String,
```

Comment:
I always run into this, so could you just document whether path includes the actual netbench scenario, of if it's just the path to the folder? I'm a big fan of including "example" members for structs like this.

Reply:
This is an intermediate that will be generated internally. The user-provided args will be introduced later and [documented](https://github.com/aws/s2n-netbench/blob/ak-fullOrchImport/netbench-orchestrator/src/orchestrator/cli.rs#L20-L34)

I think we will want to add examples to that struct.